### PR TITLE
Chore/upgrade access grants

### PR DESCRIPTION
**Pull request by VirginiaBalseiro**
## Description

Upgrade to latest access grants lib.

## Changes

- upgrading libs
- skipping a test that suddenly fails (temporary)

## Testing

- test out the flow with [access grants app ](https://test-app-access-grants.vercel.app/)

## Commit checklist

- [x] All acceptance criteria are met.
- [ ] Includes tests to ensure functionality and prevent regressions.
- [ ] Relevant documentation, if any, has been written/updated.
- [ ] The changelog has been updated, if applicable.

### CODE REVIEW

File: components/resourceDetails/resourceSharing/agentAccess/agentAccessOptionsMenu/index.test.jsx [hunk 1]
```diff
@@ -95,14 +95,15 @@ describe("Access Details Modal", () => {
     });
   });
 
-  it("closes the details modal when you click on the revoke button", async () => {
+  // FIXME: find out why this test suddenly fails
```
Comment (chelseapinka) [inline components/resourceDetails/resourceSharing/agentAccess/agentAccessOptionsMenu/index.test.jsx:98]: Shouldn't we fix this before we merge?

Comment (VirginiaBalseiro) [reply components/resourceDetails/resourceSharing/agentAccess/agentAccessOptionsMenu/index.test.jsx:98]: Yes, I just wanted it to build for testing.

Comment (VirginiaBalseiro) [reply components/resourceDetails/resourceSharing/agentAccess/agentAccessOptionsMenu/index.test.jsx:98]: fixed.

**Review by ThisIsMissEm — APPROVED** [review]
LGTM

**Pull request merged**